src/components/ProfileList/ProfileList.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import styles from '@/components/ProfileList/ProfileList.module.scss'
+import ProfilePicture from '@/assets/images/ProfilePicture.png'
+import { ProfileData } from '@/components/ProfileList/ProfileData'
+interface ProfileListItem {
+  id: number
+  name: string
+  seniority: string
+  location: string
+  technology: string
+  technology2: string
+  technology3: string
+  others: string
+}
+
+interface ProfileListProps {
+  data: ProfileListItem
+}
+
+enum JobSpecialization {
+  Frontend = 'Frontend',
+  Backend = 'Backend',
+  Fullstack = 'Fullstack',
+}
+
+interface TechnologyChipProps {
+  name: string
+  theme: JobSpecialization
+  seniority: string
+}
+
+const getTechnologyClass = (theme: JobSpecialization, seniority: string) => {
+  if (seniority.includes('Frontend')) {
+    return styles.frontend
+  } else if (seniority.includes('Backend')) {
+    return styles.backend
+  } else if (seniority.includes('Fullstack')) {
+    return styles.fullstack
+  }
+
+  switch (theme) {
+    case JobSpecialization.Frontend:
+      return styles.frontend
+    case JobSpecialization.Backend:
+      return styles.backend
+    case JobSpecialization.Fullstack:
+      return styles.fullstack
+    default:
+      return ''
+  }
+}
+
+const TechnologyChip: React.FC<TechnologyChipProps> = (props) => {
+  const themeClass = getTechnologyClass(props.theme, props.seniority)
+  return <p className={`${styles.badge} ${themeClass}`}>{props.name}</p>
+}
+
+const ProfileListItem: React.FC<ProfileListProps> = ({ data }) => {
+  let theme: JobSpecialization
```

Comment:
**nitpick:** We should use `useState` here for keeping, and updating such value: 
```
const [theme, setTheme] = useState(null)
```